addons/account/static/src/js/reconciliation/reconciliation_model.js

```diff
@@ -102,6 +102,10 @@ var StatementModel = BasicModel.extend({
         this.valuemax = 0;
     },
 
+    _round: function (value, digits) {
+        return Number(Math.round(value + 'e' + digits) + 'e-' + digits);
```

Comment:
Wouldn't it be less trashy to use `Math.pow` to shift the value back and forth around rounding?

Also beware, Math.round will round to +∞ in all cases, which is unlike pretty much anything else, and definitely unlike most (if not all) of the roundings done on the Python side.